server/repository/src/database_settings.rs

```diff
@@ -1,3 +1,4 @@
+#[cfg(all(not(feature = "postgres"), not(feature = "memory")))]
```

Comment:
offroad: noticed that when running as postgres, this shows a warning as an unused import